CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - Ruby 2.4 testing
 
+## [1.1.1] - 2017-08-01
```

Comment:
generally it's best to leave this to the maintainer who merges it to version it since there is no guarantee on merge order but in this case I am not gonna make you change it since I can get to it shortly.